### ISSUE TITLE
Reuse existing type stubs

### ISSUE THREAD

**Issue by sobolevn** [dry-python/linters#25]
There are several popular type-stubs for `django`, `drf`, `sqlalchemy`. We should reuse them.

List:
- https://github.com/mkurnikov/django-stubs
- https://github.com/mkurnikov/djangorestframework-stubs
- https://github.com/dropbox/sqlalchemy-stubs

**Comment by proofit404** [comment]
I think we should mention such integrations in the tutorials project.

What do you think?

**Comment by sobolevn** [comment]
I agree, also a demo project would be useful. Not so many people know how to setup this kind of things
